Microsoft/WebTemplateStudio
# Flask Templates for Grid Routing

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/27727

This PR adds grid Routing support using Sample Data to the Flask Application.

**How to Check**
- Generate a Flask Application
- Go the application directory and run `yarn install`
- Run 'yarn start` to run the application

- Go to /grid route to see if it is working

closes #723
